docs/11-circleci/02-activation.mdx

```diff
@@ -0,0 +1,135 @@
+---
+toc_max_heading_level: 4
+---
+
+# Activation
+
+The orb uses a Unity Editor instance to test and build your projects. These operations require the
```

Comment:
Do all GameCI users use the Unity Editor? I was wondering if it would make sense to link out to the relevant page/docs here.

Reply:
Yup. Right now, GameCI is solely focused on Unity. If users get to this documentation, we can safely assume they know what the Editor is.